command/acl/token/clone/token_clone_test.go

```diff
@@ -21,6 +21,7 @@ func parseCloneOutput(t *testing.T, output string) *api.ACLToken {
 	re := regexp.MustCompile("Token cloned successfully.\n" +
 		"AccessorID:       ([a-zA-Z0-9\\-]{36})\n" +
 		"SecretID:         ([a-zA-Z0-9\\-]{36})\n" +
+		"(?:Namespace:        default\n)?" +
```

Comment:
```suggestion
		"(?:Namespace:     default\n)?" +
```

I think there are too many spaces... not sure if I got it right in my suggestion though. hard to see.

Reply:
The spaces look off from the line above due to the extra `(?:` characters at the front of this that will not match any actual text. If you dedent it once to where the `N` lines up with the `S` in `SecretID` then the spacing appears correct.

The regex is weird so that we can expect either no line, or the `Namespace:        default` line in the case of this test being executed against the enterprise code.